components/dashboards-web-component/src/designer/components/PageEntry.jsx

```diff
@@ -94,15 +65,19 @@ export default class PageEntry extends Component {
         }
     }
 
+    handleClose() {
```

Comment:
Since the handleClose is simple function, let's move this as a lamda in line 100.